docs/en/02_Developer_Guides/15_Customising_the_Admin_Interface/How_Tos/Extend_CMS_Interface.md

```diff
@@ -13,6 +13,15 @@ simple checkbox.
 For a deeper introduction to the inner workings of the CMS, please refer to our
 guide on [CMS Architecture](/developer_guides/customising_the_admin_interface/cms_architecture).
 
+## Redux Devtools
+
+It's important to be able to view the state of the React application when you're debugging and
+building the interface.
+
+To be able to view the state, you'll need to be in a dev environment and have the redux devtools
```

Comment:
I'm not sure about providing one, as it could change at anytime, but I guess I'd suggest this one if it's better for it
https://github.com/zalmoxisus/redux-devtools-extension